src/app/pages/TokenDashboardPage/TokenDetailsCard.tsx

```diff
@@ -0,0 +1,74 @@
+import { FC } from 'react'
+import Card from '@mui/material/Card'
+import { useRequiredScopeParam } from '../../hooks/useScopeParam'
+import { useLoaderData } from 'react-router-dom'
+import { useTokenInfo } from './hook'
+import { useAccount } from '../AccountDetailsPage/hook'
+import { TextSkeleton } from '../../components/Skeleton'
+import { StyledDescriptionList } from '../../components/StyledDescriptionList'
+import { useScreenSize } from '../../hooks/useScreensize'
+import { useTranslation } from 'react-i18next'
+import { AccountLink } from '../../components/Account/AccountLink'
+import { CopyToClipboard } from '../../components/CopyToClipboard'
+import { ContractVerificationIcon } from '../../components/ContractVerificationIcon'
+import { getTokenTypeName } from './TokenTypeCard'
+import { getNameForTicker, Ticker } from '../../../types/ticker'
+import { ContractCreatorInfo } from '../../components/Account/ContractCreatorInfo'
+
+export const TokenDetailsCard: FC = () => {
+  const { t } = useTranslation()
+  const scope = useRequiredScopeParam()
+  const address = useLoaderData() as string
+  const { isMobile } = useScreenSize()
+
+  const { token, isLoading: tokenIsLoading } = useTokenInfo(scope, address)
+  const { account, isLoading: accountIsLoading } = useAccount(scope, address)
+  const isLoading = tokenIsLoading || accountIsLoading
+
+  const contract = account?.evm_contract
+
+  const transactionsLabel = account ? account.stats.num_txns.toLocaleString() : ''
+  const balance = account?.balances[0]?.balance || '0'
```

Comment:
This seems unsafe

Reply:
OK, fixed it. We will just display n/a when there is no balance. Also fixed it in Account, where we had the same.